optuna/study/study.py

```diff
@@ -1091,6 +1091,7 @@ def _should_skip_enqueue(self, params: Mapping[str, JSONSerializable]) -> bool:
                     if isinstance(param_value, Real)
                     else param_value == existing_param
                 )
+                assert isinstance(is_repeated, bool)
```

Comment:
This can be `False`.  How about `repeated_params: list[bool | np.bool] = []` or `bool(is_repeated)`?

Reply:
Thank you. I applied your approach.